app/src/components/domain/BaseMap/index.tsx

```diff
@@ -104,10 +101,9 @@ function BaseMap(props: Props) {
         [countries],
     );
 
-    const {
-        currentLanguage,
-    } = useContext(LanguageContext);
+    // const { currentLanguage } = useContext(LanguageContext);
```

Comment:
Why we comment it out? Why don't we just remove it?

Reply:
I have removed these!